moto/resourcegroupstaggingapi/models.py

```diff
@@ -310,6 +315,39 @@ def format_tag_keys(
 
                 yield {"ResourceARN": f"{api.arn}", "Tags": tags}
 
+        # Athena
+        if self.athena_backend:
+            if not resource_type_filters or "athena" in resource_type_filters:
+                athena_backend = athena_backends[self.account_id][self.region_name]
+
+                # Capacity Reservations
+                for (
+                    capacity_reservation
+                ) in self.athena_backend.capacity_reservations.values():
+                    tags = athena_backend.tagger.list_tags_for_resource(
+                        capacity_reservation.name
```

Comment:
I don't understand the difference between your use of `athena_backend` vs `self.athena_backend`... what am I missing?

Reply:
It should have been `athena_backend`. Updated!